.github/workflows/test.yml

```diff
@@ -24,10 +24,7 @@ on:
   workflow_dispatch:
 
 env:
-  ETCDCTL_API: "3"
   PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION: python
-  ETCD_DOWNLOAD_VERSION: "3.4.24"
-  CONSUL_VERSION: "1.14.4"
```

Comment:
By having them as matrix inputs, we get these to show up like below, and we get the versions testable job-by-job (used for traefik-version).

![image](https://github.com/jupyterhub/traefik-proxy/assets/3837114/f026f697-3597-4ad6-9d10-89e2331c4ef6)